pkgs/top-level/all-packages.nix

```diff
@@ -11376,7 +11376,6 @@ in
     frei0r = if stdenv.isDarwin then null else frei0r;
     game-music-emu = if stdenv.isDarwin then null else game-music-emu;
     libjack2 = if stdenv.isDarwin then null else libjack2;
-    libvmaf = if stdenv.isDarwin then null else libvmaf;
```

Comment:
Actually - if I remember correctly, this was needed to avoid build errors? Why remove it?

Reply:
yes, but only for darwin. I've tested the new version on darwin and it works